### DISCUSSION TITLE
Higher Output Samplerates

### DISCUSSION THREAD

**Discussion by Nexeration** [Anjok07/ultimatevocalremovergui#694] [Ideas]
I really love the models and everything but it would be great to export in 48khz instead of only 44,1khz. 
I think there is no way to adjust that at the moment? Would be great for better use cases in DAWs like Ableton ;)

**Comment by Githubfan10** [comment]
waiting for answers as well

**Comment by paulzinman** [comment]
Agree. What this software does is great, but the majority of my work is 96kHz files. It appears that any input sample rate is handled automatically, but it would be great to be able select any standard sample rate (up to 192kHz at least) for output (or at the vary least, to have an option for the output sample rate to match the input). Is the issue that all the models are trained at 44.1kHz?
